### ISSUE TITLE
Wikipedia file (Zim) size has apparently decreased. After deleting some other files in the old download directory, Wikipedia has disappeared from "Local files" and needs to be redownloaded

### ISSUE THREAD

**Issue by FluffyCatCuddlyCat** [kiwix/kiwix-desktop#563]
I downloaded a Wikipedia (_English - nopic - Dated: 2020-10-02 - 39 GB_), and kiwix was able to successfully open it for a few days. However, after some days, kiwix started crashing (_"kiwix has stopped responding"_) whenever I tried to open the Wikipedia. I noticed that, somehow, the Wikipedia file size in the download folder (file name: _wikipedia_en_all_nopic_2020-10.zim_) has changed to 29.2 GB (it was around 39 GB previously).

My current download location is different from the default one. The default download location had some small files in it (I am not sure but I believe the files were less than 10 MB total). I deleted those files (for no particular reason). Now, the said Wikipedia does not appear under "Local Files" in kiwix. When I try to download it, it starts from the beginning.

Basically, I am not aware of any changes I might have made that could have caused the change in the Wikipedia file size. I would really appreciate it if the lost portion of the file could be locally recovered or if I don't have to redownload the complete file (since downloading 39 GB takes nearly a week for me).

Kiwix version: 2.0.5 (64 bit) - Windows 10

Any insights or suggestions are most welcomed and appreciated.

**Comment by kelson42** [comment]
@FluffyCatCuddlyCat Thank you for your bug report. Would you be able to upload the corrupted zim file somewhere? So we can have a look?

**Comment by FluffyCatCuddlyCat** [comment]
@kelson42 No, sorry. That would take a lot of time because my upload speed is around 0.2 MBps and the file is 29.2 GB. Actually, the issue can probably be solved by redownloading the file from kiwix but I would rather avoid doing that if possible.

**Comment by kelson42** [comment]
From what I understand, for some reason your ZIM file has been corrupted/truncated. This is not something that Kiwix does. For the moment we have no solution for incremental update. You will have to redownload the ZIM file fully. Because we can not do anything more on this ticket, I will close it.
